workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/loadControl/loadPath/SyncedFindMaxLoadPath.java

```diff
@@ -234,15 +242,18 @@ private UniformLoadInterval getNextInitialRampInterval(boolean prevIntervalPasse
 		logger.debug("getNextInitialRampInterval " + this.getName() + ": interval " + intervalNum);
 
 		/*
-		 * If we have reached max users, or the previous interval failed, then to go to
+		 * If the previous interval failed, then to go to
 		 * the FINDFIRSTMAX phase
 		 */
-		if (!prevIntervalPassed || ((curUsers + getInitialRampRateStep()) > maxUsers)) {
```

Comment:
Is this check no longer needed in the non hint case also?

Reply:
This check was moved to the if clause that you commented on in initialRampIntervalComplete(), so it isn't needed here.